pyk/src/pyk/ktool/krun.py

```diff
@@ -36,6 +36,7 @@ class KRunOutput(Enum):
     LATEX = 'latex'
     KORE = 'kore'
     NONE = 'none'
+    HINTS = 'hints'
```

Comment:
Now that there's `run_proof_hint_process`, this seems unnecessary.

Reply:
I prefer no new value. Any new value breaks `_krun`.

Reply:
Deleted